step-core-data/src/main/resources/com/tyndalehouse/step/core/data/create/lexicon/lexicon_greek.txt

```diff
@@ -62370,7 +62370,7 @@ $=G2424I=ιησους==============
 @StrBetaAccented=	*)ihsou=s
 @StrTranslit=	Iēsoûs
 @StrPronunc=	ee-ay-sooce'
-@Translations=	Jesus/Joshua
+@Translations=	Joshua
```

Comment:
Do we want the "translations" field to only be one word?

Reply:
David and another person will make the changes for many other words.